### DISCUSSION TITLE
eForms SDK 1.12 release date

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#968] [Q&A] [answered]
Do you know when the eForms SDK 1.12 will be released?

Thanks in advance,
Alex

**Comment by bertrand-lorentz** [comment, marked as answer]
We have just tagged the second release candidate for SDK 1.12.0: https://github.com/OP-TED/eForms-SDK/releases/tag/1.12.0-rc.2

We are testing that it does not have any remaining problem, which might take a few days. If it's OK, we will then publish the 1.12.0 release.
